docs/snippets/react/button-story-click-handler-simple-docs.js.mdx

```diff
@@ -1,5 +1,17 @@
 ```js
 // Button.stories.js | Button.stories.jsx | Button.stories.ts | Button.stories.tsx
 
-export const Text = ({ label, onClick }) => <Button label={label} onClick={onClick} />;
-```
\ No newline at end of file
+import React from 'react';
+
+import { action } from '@storybook/addon-actions';
```

Comment:
is this action used?

Reply:
This particular snippet was converted as a safeguard. It's used in this [section](https://storybook.js.org/docs/react/api/csf#args-story-inputs)! That page will be redone and that snippet might even disappear. As we're not adding anything new with this snippet.